mfkdf2/src/derive/factors/totp.rs

```diff
@@ -170,7 +170,6 @@ pub fn totp(code: u32, options: Option<TOTPDeriveOptions>) -> MFKDF2Result<MFKDF
       code,
       target: 0,
     }),
-    salt:        [0u8; 32].to_vec(),
     entropy:     Some(0.0), // TODO (@lonerapier): is entropy used anywhere after derive?
```

Comment:
re: comment

No, and it's dangerous to store it in a policy. This is why we should also consider a type state for `Derived` so that you cannot possibly write entropy to the policy. 

This is also an overkill way of doing it, but food for thought.